rust/src/measurements/make_private_lazyframe/group_by/pseudocode/make_private_group_by.py

```diff
@@ -12,18 +12,18 @@ def make_private_group_by(
     t_prior = input_expr.make_stable(input_domain, input_metric)  # |\label{line:tprior}|
     middle_domain, middle_metric = t_prior.output_space()
 
-    grouping_columns = match_grouping_columns(keys)  # |\label{line:groupcols}|
+    by = match_grouping_columns(keys)  # |\label{line:groupcols}|
```

Comment:
Very slight preference to keep the longer name for clarity.

Reply:
I'm going to hit this again before too long with expression grouping keys and expressions in margins.